electron/renderer/static/webview-preload.js

```diff
@@ -150,7 +150,7 @@ const replaceGoogleAuth = () => {
 };
 
 const enableFileLogging = () => {
-  const id = new URL(window.location).searchParams.get('id');
+  const id = new URL(window.location.href).searchParams.get('id');
```

Comment:
Actual fix

Reply:
Let's move part of this complicated structure out into a const